tox.ini

```diff
@@ -1,12 +1,13 @@
 [tox]
-envlist = py{27,34,35}-django{18,19,master},qa
+envlist = py{27,35,36}-dj{18,19,master},qa
 
 [testenv]
 deps=
     -rrequirements-dev.txt
-    django18: Django>=1.8,<1.9
```

Comment:
This might make testing slower as PyPI packages are cached and installed very fast in pip caches whereas tarballs and zip installs from GitHub repositories might be very slow to download and won't be cached in any manner.